src/view/pages/home/main/HomeMain.tsx

```diff
@@ -58,7 +59,9 @@ const HomeMain = () => {
 						<MainCard key={statement.statementId} statement={statement} />
 					))
 				) : (
-					<PeopleLoader />
+					<div className="peopleLoadingScreen">
+						<PeopleLoader />
+					</div>
```

Comment:
You don't have to wrap the component in a div. 
Find another way to relocate the loader to the middle of the screen.

Reply:
Thank you for the feedback.
The component cannot accept classname or style.. Should I do something else?